kratos/utilities/os_utilities.cpp

```diff
@@ -41,5 +44,132 @@ std::string GetCurrentWorkingDir()
     KRATOS_CATCH("")
 }
 
+/***********************************************************************************/
+/***********************************************************************************/
+
+void RemoveFile(const std::string& rFileName)
+{
+    const bool is_file = IsFile(rFileName);
+    if (is_file) { // It is a file
+        const char* name = rFileName.c_str();
+        remove(name);
+    } else {
+        KRATOS_WARNING("OSUtilities") << "This is a folder. Please use RemoveDir" << std::endl;
+    }
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void RemoveFileInCurrentWorkingDir(const std::string& rFileName)
+{
+    RemoveFile(JoinPath(GetCurrentWorkingDir(), rFileName));
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void RemoveDir(const std::string& rFolderName)
+{
+    const bool is_dir = IsDir(rFolderName);
+
+    if (is_dir) { // It is a directory
+        const char* name = rFolderName.c_str();
+#ifdef KRATOS_COMPILED_IN_WINDOWS
+        RemoveDirectoryA(name);
+#else
+        rmdir(name);
+#endif
+    } else {
+        KRATOS_WARNING("OSUtilities") << "This is a file. Please use RemoveFile" << std::endl;
```

Comment:
```suggestion
        KRATOS_WARNING("OSUtilities") << "\"" << rFolderName << "\" could not be removed because it is not a folder" << std::endl;
```

Reply:
you cannot say that it is a file, could also be a link or sth else